projects/angular/src/data/tree-view/tree-node.html

```diff
@@ -65,8 +65,8 @@
   <ng-template #treenodeContent>
     <ng-content></ng-content>
     <div class="clr-sr-only" *ngIf="featuresService.selectable || ariaSelected">
-      <span *ngIf="ariaSelected"> selected</span>
-      <span *ngIf="!ariaSelected"> unselected</span>
+      <span *ngIf="ariaSelected"> {{selectedString}} </span>
+      <span *ngIf="!ariaSelected"> {{unselectedString}} </span>
```

Comment:
Isn't is better this way?
No need to add geters as well.
```suggestion
      <span>{{ariaSelected ? commonStrings.keys.selectedTreeNode : commonStrings.keys.unselectedTreeNode}}</span>
```

Reply:
yes, in my opinion looks better that way

Reply:
Updated in https://github.com/vmware-clarity/ng-clarity/pull/1575/commits/a35bf2586d73d31cf9cb274b679f3f61fc90d45f